add outernet telemetry client and service

abgoyal opened the pull request.

Commit:
add outernet telemetry client and service

abgoyal committed:
update to latest release of telemetry_client

foxbunny commented on telemetry/Config.in:
```diff
@@ -0,0 +1,8 @@
+menuconfig BR2_PACKAGE_TELEMETRY
```
This should be `config` since there are no extra options

foxbunny commented on telemetry/S91telemetry:
```diff
@@ -0,0 +1,36 @@
+        [ -f $TELEMETRY_PID ] && exit 0
+
+        printf "Starting telemetry: "
+        start-stop-daemon -S -q -p $TELEMETRY_PID -x $DAEMON &
```
Maybe here it can use the -b (--background) switch instead of &?

foxbunny commented on telemetry/telemetry.mk:
```diff
@@ -0,0 +1,16 @@
+TELEMETRY_SETUP_TYPE = setuptools
+
+define TELEMETRY_INSTALL_INIT_SYSV
+	$(INSTALL) -Dm0755 $(BR2_EXTERNAL)/package/telemetry/S91telemetry \
```
`$(BR2_EXTERNAL)/package/telemetry` can be replaced with `$(call epkgdir,telemetry)`

Commit:
Use config instead of menuconfig

Since there are no extra options, config is used instead of menucofnig to
reduce the visual noise cause by menuconfig.

Signed-off-by: Branko Vukelic <branko@outernet.is>

Commit:
Use -b and -m switches, clean up PID file

This patch fixes the init script so it uses -b switch instead of manually
backgrounding the proc, and also adds the -m switch so that the PID file is
actually written. In the stop section, the PID file is also cleaned up as
start-stop-daemon apparently won't do it for us.

Signed-off-by: Branko Vukelic <branko@outernet.is>

Commit:
Use epkgdir shortcut instead of BR2_EXTERNAL path

Signed-off-by: Branko Vukelic <branko@outernet.is>
